client-spark/spark3/src/main/java/org/apache/spark/shuffle/reader/RssShuffleReader.java

```diff
@@ -114,7 +115,7 @@ public RssShuffleReader(
     this.taskIdBitmap = taskIdBitmap;
     this.hadoopConf = hadoopConf;
     this.readMetrics = readMetrics;
-    this.partitionToShuffleServers = rssShuffleHandle.getPartitionToServers();
+    this.partitionToShuffleServers = allPartitionToServers;
```

Comment:
Here too, can we put the information maintained by ShuffleManagerService into rssShuffleHandle, so that it is transparent to RssShuffleReader and RssShuffleWriter.